### ISSUE TITLE
[New device support]: TuYa 24GHz Human Presence Sensor TS0225_TZE200_2aaelwxk

### ISSUE THREAD

**Issue by chewth91** [Koenkk/zigbee2mqtt#18612]
### Link

https://www.aliexpress.com/item/1005005909540714.html?gps-id=pcStoreNewArrivals&scm=1007.23409.271287.0&scm_id=1007.23409.271287.0&scm-url=1007.23409.271287.0&pvid=5175abbe-b739-46df-8ec7-c2c2a8f4c801&_t=gps-id:pcStoreNewArrivals,scm-url:1007.23409.271287.0,pvid:5175abbe-b739-46df-8ec7-c2c2a8f4c801,tpp_buckets:668%232846%238114%231999&pdp_npi=4%40dis%21MYR%21107.42%2175.21%21%21%2122.84%21%21%402101c59816919999368335919e0b96%2112000034805127430%21rec%21MY%21%21A&spm=a2g0o.store_pc_home.smartNewArrivals_2002856433204.1005005909540714

### Database entry

{"id":67,"type":"Router","ieeeAddr":"0xa4c138f81e577a0c","nwkAddr":64592,"manufId":4417,"manufName":"_TZE200_2aaelwxk","powerSource":"DC Source","modelId":"TS0225","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":1026,"inClusterList":[0,3,1280,57346,61184,60928,57344,1024],"outClusterList":[],"clusters":{"genBasic":{"attributes":{"zclVersion":3}},"ssIasZone":{"attributes":{"iasCieAddr":"0x00124b00257cc7da","zoneState":0}},"msIlluminanceMeasurement":{"attributes":{"measuredValue":32770}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":130,"stackVersion":2,"hwVersion":1,"swBuildId":"0122052017","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1691999105436,"defaultSendRequestWhen":"immediate"}

### Comments

Hi, just got this device from aliexpress recently and found that it is not supported in Z2M. 

### External converter

```shell
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const legacy = require('zigbee-herdsman-converters/lib/legacy');
const extend = require('zigbee-herdsman-converters/lib/extend');
const ota = require('zigbee-herdsman-converters/lib/ota');
const tuya = require('zigbee-herdsman-converters/lib/tuya');
const utils = require('zigbee-herdsman-converters/lib/utils');
const globalStore = require('zigbee-herdsman-converters/lib/store');
const e = exposes.presets;
const ea = exposes.access;

const tzDatapoints = {
    ...tuya.tz.datapoints,
    key: [...tuya.tz.datapoints.key, 'alarm', 'alarm_time', 'alarm_volume', 'type', 'volume', 'ringtone', 'duration','indicator',
    'motion_detection_distance', 'motion_detection_sensitivity', 'small_motion_detection_distance',
    'small_motion_detection_sensitivity', 'static_detection_distance', 'static_detection_sensitivity','motion_false_detection','fading_time',
    'breathe_false_detection','working_mode']
}

const definition = {
    fingerprint: tuya.fingerprint('TS0225', ['_TZE200_2aaelwxk']),
    model: 'ZG-205Z/A',
    vendor: 'TuYa',
    description: '24Ghz Human presence sensor',
    fromZigbee: [tuya.fz.datapoints],
    toZigbee: [tzDatapoints],
    exposes: [
        e.enum('presence_state', ea.STATE, ['none', 'presence']).withDescription('Presence'),
        e.enum('human_motion_state', ea.STATE, ['none', 'large', 'small', 'breathe']).withDescription('Human Motion state'),
        e.numeric('illuminance_value', ea.STATE).withDescription('Illuminance').withUnit('lux'),            
        e.numeric('fading_time', ea.STATE_SET).withValueMin(0).withValueMax(28800).withValueStep(1).withUnit('s')
            .withDescription('Presence keep time'),
        e.numeric('motion_detection_distance', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(0.01).withUnit('m')
            .withDescription('Motion detection distance'),
        e.numeric('motion_detection_sensitivity', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(1).withUnit('x')
            .withDescription('Motion detection sensitivity'),
        e.numeric('small_motion_detection_distance', ea.STATE_SET).withValueMin(0).withValueMax(6).withValueStep(0.01).withUnit('m')
            .withDescription('Small motion detection distance'),
        e.numeric('small_motion_detection_sensitivity', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(1).withUnit('x')
            .withDescription('Small motion detection sensitivity'),
        e.numeric('static_detection_distance', ea.STATE_SET).withValueMin(0).withValueMax(6).withValueStep(0.01).withUnit('m')
            .withDescription('Static detection distance'),
        e.numeric('static_detection_sensitivity', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(1).withUnit('x')
            .withDescription('Static detection sensitivity'),
        e.numeric('motion_false_detection', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(1).withUnit('x')
            .withDescription('Motion false detection'),            
        e.binary('indicator', ea.STATE_SET, 'ON', 'OFF').withDescription('LED Indicator'),
        e.binary('breathe_false_detection', ea.STATE_SET, 'ON', 'OFF').withDescription('Breathe False Detection'),
    ],
    meta: {
        tuyaDatapoints: [
            [1, 'presence_state', tuya.valueConverterBasic.lookup({'none': tuya.enum(0), 'presence': tuya.enum(1)})],
            [2, 'motion_detection_sensitivity', tuya.valueConverter.raw],
            [3, 'mov_minimum_distance', tuya.valueConverter.raw],
            [4, 'motion_detection_distance', tuya.valueConverter.divideBy100],
            [101, 'human_motion_state', tuya.valueConverterBasic.lookup({'none': tuya.enum(0), 'large': tuya.enum(1), 'small': tuya.enum(2), 'breathe': tuya.enum(3)})],
            [102, 'fading_time', tuya.valueConverter.raw],
            [103, 'motion_false_detection', tuya.valueConverter.raw],
            [104, 'small_motion_detection_distance', tuya.valueConverter.divideBy100],
            [105, 'small_motion_detection_sensitivity', tuya.valueConverter.raw],
            [106, 'illuminance_value', tuya.valueConverter.raw],
            [107, 'indicator', tuya.valueConverter.onOff],
            [108, 'static_detection_distance', tuya.valueConverter.divideBy100],
            [109, 'static_detection_sensitivity', tuya.valueConverter.raw],
            [110, 'micro_minimum_distance', tuya.valueConverter.raw],
            [111, 'motionless_minimum_distance', tuya.valueConverter.raw],
            [112, 'reset_setting', tuya.valueConverter.raw],
            [113, 'breathe_false_detection', tuya.valueConverter.raw],
            [114, 'time', tuya.valueConverter.raw],      
            [115, 'alarm_time', tuya.valueConverter.raw],            
            [116, 'alarm_volume', tuya.valueConverterBasic.lookup({'low': tuya.enum(0), 'medium': tuya.enum(1), 'high': tuya.enum(2), 'mute': tuya.enum(3)})],
            [117, 'working_mode', tuya.valueConverterBasic.lookup({'arm': tuya.enum(0), 'off': tuya.enum(1), 'alarm': tuya.enum(2),  'doorbell': tuya.enum(3)})],
            [118, 'auto1', tuya.valueConverter.raw],      
            [119, 'auto2', tuya.valueConverter.raw],      
            [120, 'auto3', tuya.valueConverter.raw],                  
        ],
    },
};

module.exports = definition;
```

**Comment by Koenkk** [comment]
Does this ext converter work? I yes, could you make a pull request?

**Comment by chewth91** [comment]
> Does this ext converter work? I yes, could you make a pull request?

@Koenkk it's working fine for and haven't seen any issues. I am not familiar with GitHub and am not sure about how to make a pull request.

**Comment by kkossev** [comment]
I can confirm that Tuya DPs shown above are the correct ones for TS0225 _TZE200_2aaelwxk

@chewth91 I suppose that you have Tuya Zigbee GW, are there "Move Self Test", "Small Move Self Test" and "Breathe Self Test" functions in the Smart Life app?   ( these could be potentially DP 118, 119, and 120).

**Comment by chewth91** [comment]
> I can confirm that Tuya DPs shown above are the correct ones for TS0225 _TZE200_2aaelwxk
> 
> @chewth91 I suppose that you have Tuya Zigbee GW, are there "Move Self Test", "Small Move Self Test" and "Breathe Self Test" functions in the Smart Life app?   ( these could be potentially DP 118, 119, and 120).

Hi, i did not see self test in Smartlife. 


![Screenshot_20230819-015524.png](https://github.com/Koenkk/zigbee2mqtt/assets/29686018/c7c8d2b1-d02e-44fe-a443-a4c0d1e6dc3d)

![Screenshot_20230819-015702.png](https://github.com/Koenkk/zigbee2mqtt/assets/29686018/857f0d1b-3b65-4dc7-9468-b1ad1e74e170)

![Screenshot_20230819-015724.png](https://github.com/Koenkk/zigbee2mqtt/assets/29686018/f9b125c8-93dd-4f10-bd16-12987c2aad4a)

![Screenshot_20230819-015733.png](https://github.com/Koenkk/zigbee2mqtt/assets/29686018/508eb3e4-5be0-4213-93a3-603526f7d0be)

**Comment by Koenkk** [comment]
All `TS0225` will be detected as supported now.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by leroyloren** [comment]
I have **_TZE200_2aaelwxk** with this converter it detects but doesn't show **motion_state** and **illuminance_lux**, the rest seems to work.

I am attaching debug logs etc.

![obrazek](https://github.com/Koenkk/zigbee2mqtt/assets/57643470/61e46eab-1776-480a-a572-98425e806511)

`{"id":41,"type":"Router","ieeeAddr":"0xa4c1388baef32d03","nwkAddr":34496,"manufId":4417,"manufName":"_TZE200_2aaelwxk","powerSource":"DC Source","modelId":"TS0225","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":1026,"inClusterList":[0,3,1280,57346,61184,60928,57344,1024],"outClusterList":[],"clusters":{"genBasic":{"attributes":{}},"ssIasZone":{"attributes":{"iasCieAddr":"0x00124b0021b9518e","zoneState":0}},"msIlluminanceMeasurement":{"attributes":{"measuredValue":33545}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":130,"stackVersion":2,"hwVersion":1,"swBuildId":"0122052017","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1692383344135,"defaultSendRequestWhen":"immediate"}`

![obrazek](https://github.com/Koenkk/zigbee2mqtt/assets/57643470/f365ec35-eec0-479f-86b3-1d53782519c3)

![obrazek](https://github.com/Koenkk/zigbee2mqtt/assets/57643470/2d46a3bc-364e-455d-837c-d695c753e8c9)


https://www.aliexpress.com/item/1005005904142527.html

![Tuya-Zigbee-Human-Presence-Detector-Smart-Human-Body-PIR-Sensor-Radar-DetectorMotion-Sensors-Support-Home-Assistant jpg_640x640](https://github.com/Koenkk/zigbee2mqtt/assets/57643470/8e838dfb-1945-46de-a3f9-7c6988f27192)

**Comment by kkossev** [comment]
@chewth91 thank you for the screenshots - obviously, this model does not have exactly the same functions as the other TS0225 radar :

![image](https://github.com/Koenkk/zigbee2mqtt/assets/6189950/fe3ab6b7-ab7b-41c0-8db3-8f3b80a1aebf)

**Comment by chewth91** [comment]
> I have **_TZE200_2aaelwxk** with this converter it detects but doesn't show **motion_state** and **illuminance_lux**, the rest seems to work.
> 
> I am attaching debug logs etc.
> 
> ![obrazek](https://user-images.githubusercontent.com/57643470/261687703-61e46eab-1776-480a-a572-98425e806511.png)
> 
> `{"id":41,"type":"Router","ieeeAddr":"0xa4c1388baef32d03","nwkAddr":34496,"manufId":4417,"manufName":"_TZE200_2aaelwxk","powerSource":"DC Source","modelId":"TS0225","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":1026,"inClusterList":[0,3,1280,57346,61184,60928,57344,1024],"outClusterList":[],"clusters":{"genBasic":{"attributes":{}},"ssIasZone":{"attributes":{"iasCieAddr":"0x00124b0021b9518e","zoneState":0}},"msIlluminanceMeasurement":{"attributes":{"measuredValue":33545}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":130,"stackVersion":2,"hwVersion":1,"swBuildId":"0122052017","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1692383344135,"defaultSendRequestWhen":"immediate"}`
> 
> ![obrazek](https://user-images.githubusercontent.com/57643470/261688040-f365ec35-eec0-479f-86b3-1d53782519c3.png)
> 
> ![obrazek](https://user-images.githubusercontent.com/57643470/261688726-2d46a3bc-364e-455d-837c-d695c753e8c9.png)
> 
> https://www.aliexpress.com/item/1005005904142527.html
> 
> ![Tuya-Zigbee-Human-Presence-Detector-Smart-Human-Body-PIR-Sensor-Radar-DetectorMotion-Sensors-Support-Home-Assistant jpg_640x640](https://user-images.githubusercontent.com/57643470/261687219-8e838dfb-1945-46de-a3f9-7c6988f27192.jpg)

This happened to me before as well. Have you entered all the parameters?

**Comment by Koenkk** [comment]
I've adapter the converter for the `_TZE200_2aaelwxk`, can you test if everything works? 

https://gist.github.com/Koenkk/581a5f396d1ca9ac0ea9477b1017af36

**Comment by alexvaltchev** [comment]
@Koenkk when will this sensor be fully operational and working in regular Z2M ? I tried this external converter for **"TS0225 by _TZE200_2aaelwxk"** version, and it shows, but none of the sliders seem to be working. Am I doing anything wrong? OR is there better way to work in ZHA perhaps? Have anyone made this sensor working? Thanks

ZHA:
![Screenshot_1](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/f641d08b-a8db-4cc6-89e1-dd1b4cdef45b)

Z2M:
![Screenshot_2](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/6b86798b-b9e9-41c0-9135-7cf664a5a081)
![Screenshot_3](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/a808abf1-10da-488a-aca6-bc1ea13ef008)
![Screenshot_4](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/1d8b938e-2012-48d6-a84d-28ae97bb4026)

Is this Zigbee2MQTT fully functional now?

Does anyone know what each of those settings mean and how to set them for best results?
Thanks

**Comment by Koenkk** [comment]
@13717033460 since you added support for this, can you check the issue above?

**Comment by alexvaltchev** [comment]
I contacted the AliExpress seller that sent me the sensor and they have emailed me their converter. It is different. Can anybody check both and let me know which one is better and should be used?

Here is the link:[]( https://doc.szalarm.com/zg-205Z/z2m.html)

I also asked what each setting do, and here is their link with information: 
LINK: [](https://doc.szalarm.com/zg-205Z/en/)

**Comment by alexvaltchev** [comment]
@13717033460 so the AliExpress seller converter is the latest and better?

**Comment by 13717033460** [comment]
sorry,I made a mistake. This is it. Please check the attachment.



------------------&nbsp;Original&nbsp;------------------
From:                                                                                                                        "Koenkk/zigbee2mqtt"                                                                                    ***@***.***&gt;;
Date:&nbsp;Tue, Aug 29, 2023 07:59 PM
***@***.***&gt;;
***@***.******@***.***&gt;;
Subject:&nbsp;Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz Human Presence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
@13717033460 so the AliExpress seller converter is the latest and better?

**Comment by 13717033460** [comment]
{
&nbsp; &nbsp; &nbsp; &nbsp; fingerprint: tuya.fingerprint('TS0225', ['_TZE200_2aaelwxk']),
&nbsp; &nbsp; &nbsp; &nbsp; model: 'ZG-205Z-A',
&nbsp; &nbsp; &nbsp; &nbsp; vendor: 'TuYa',
&nbsp; &nbsp; &nbsp; &nbsp; description: 'Mini 24Ghz human presence sensor',
&nbsp; &nbsp; &nbsp; &nbsp; fromZigbee: [tuya.fz.datapoints],
&nbsp; &nbsp; &nbsp; &nbsp; toZigbee: [tzDatapoints],
&nbsp; &nbsp; &nbsp; &nbsp; exposes: [
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.presence(),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.enum('motion_state', ea.STATE, ['none', 'large', 'small', 'static']).withDescription('Motion state'),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.illuminance_lux(),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.numeric('fading_time', ea.STATE_SET).withValueMin(0).withValueMax(600).withValueStep(1).withUnit('s')
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; .withDescription('Presence keep time'),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.numeric('large_motion_detection_distance', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(0.01).withUnit('m')
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; .withDescription('Large motion detection distance'),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.numeric('large_motion_detection_sensitivity', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(1).withUnit('x')
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; .withDescription('Large motion detection sensitivity'),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.numeric('small_motion_detection_distance', ea.STATE_SET).withValueMin(0).withValueMax(6).withValueStep(0.01).withUnit('m')
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; .withDescription('Small motion detection distance'),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.numeric('small_motion_detection_sensitivity', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(1).withUnit('x')
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; .withDescription('Small motion detection sensitivity'),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.numeric('static_detection_distance', ea.STATE_SET).withValueMin(0).withValueMax(6).withValueStep(0.01).withUnit('m')
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; .withDescription('Static detection distance'),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.numeric('static_detection_sensitivity', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(1).withUnit('x')
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; .withDescription('Static detection sensitivity'),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; e.binary('indicator', ea.STATE_SET, 'ON', 'OFF').withDescription('LED indicator mode'),
&nbsp; &nbsp; &nbsp; &nbsp; ],
&nbsp; &nbsp; &nbsp; &nbsp; meta: {
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; tuyaDatapoints: [
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [1, 'presence', tuya.valueConverter.trueFalse1],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [106, 'illuminance_lux', tuya.valueConverter.raw],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [101, 'motion_state', tuya.valueConverterBasic.lookup({
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; 'none': tuya.enum(0), 'large': tuya.enum(1), 'small': tuya.enum(2), 'static': tuya.enum(3),
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; })],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [102, 'fading_time', tuya.valueConverter.raw],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [4, 'large_motion_detection_distance', tuya.valueConverter.divideBy100],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [2, 'large_motion_detection_sensitivity', tuya.valueConverter.raw],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [104, 'small_motion_detection_distance', tuya.valueConverter.divideBy100],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [105, 'small_motion_detection_sensitivity', tuya.valueConverter.raw],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [108, 'static_detection_distance', tuya.valueConverter.divideBy100],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [109, 'static_detection_sensitivity', tuya.valueConverter.raw],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [107, 'indicator', tuya.valueConverter.onOff],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; ],
&nbsp; &nbsp; &nbsp; &nbsp; },
&nbsp; &nbsp; },





------------------&nbsp;Original&nbsp;------------------
From:                                                                                                                        "Koenkk/zigbee2mqtt"                                                                                    ***@***.***&gt;;
Date:&nbsp;Tue, Aug 29, 2023 08:20 PM
***@***.***&gt;;
***@***.******@***.***&gt;;
Subject:&nbsp;Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz Human Presence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
Sorry, am I blind? I dont see attachment. Do you mean the converter that is posted by @chewth91 is the latest and best? OR the seller from AliExpress is newer and better? I feel stupid :( I am so sorry to ask again...

**Comment by alexvaltchev** [comment]
Thank you. Will try it now :)
What is new/different than the AliExpress version, do you know? Does it work better or more accurate? Thanks

**Comment by 13717033460** [comment]
OK,
In the process of using the product, you can contact me if you have any other questions, we are the manufacturer.

发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Tue,Aug 29,2023 8:29 PM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
Thank you. Will try it now :)
 What is new/different than the AliExpress version, do you know? Does it work better or more accurate? Thanks

**Comment by alexvaltchev** [comment]
Great, I just want the latest external converter that support TS0225_TZE200_2aaelwxk version. I dont see attached file or link that I can download it. Thanks

**Comment by 13717033460** [comment]
Yes, this one is 5.8G, but the algorithm is much better than the normal 5.8G

发自我的iPhone


------------------ Original ------------------
From: kkossev ***@***.***&gt;
Date: Tue,Aug 29,2023 11:29 PM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





  
if you have any other questions, we are the manufacturer.
  
Hi @13717033460 ,
 
The first question is about what mmWave module is actually inside TS0225_TZE200_2aaelwxk
 
 
This is not a 24Ghz module as advertised, but a 5.8 Ghz mmWave module - can you confirm that?
 
Following the AliExpress seller link in the first post by @chewth91 , we see it has changed - there is no "24 GHz" anymore on the product description. Other AE sellers removed the ''24GHz" from the description, but the wrong info can still be seen in some of the pictures.
 
Can you comment on this situation?

**Comment by 13717033460** [comment]
Sorry! &nbsp; AE seller advertising is wrong, long ago notified them to change



发自我的iPhone


------------------ Original ------------------
From: kkossev ***@***.***&gt;
Date: Tue,Aug 29,2023 11:29 PM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





  
if you have any other questions, we are the manufacturer.
  
Hi @13717033460 ,
 
The first question is about what mmWave module is actually inside TS0225_TZE200_2aaelwxk
 
 
This is not a 24Ghz module as advertised, but a 5.8 Ghz mmWave module - can you confirm that?
 
Following the AliExpress seller link in the first post by @chewth91 , we see it has changed - there is no "24 GHz" anymore on the product description. Other AE sellers removed the ''24GHz" from the description, but the wrong info can still be seen in some of the pictures.
 
Can you comment on this situation?

**Comment by kkossev** [comment]
Thank you for the answer, @13717033460 , fair enough. I appreciate your honesty.
The AE sellers bad practices are another topic, I don't want to hijack this technical thread.

**Comment by alexvaltchev** [comment]
I checked on AliExpress most sellers say 24G:

https://www.aliexpress.com/item/1005005919353153.html
https://www.aliexpress.com/item/1005005904142527.html
https://www.aliexpress.com/item/1005005914116739.html

**Comment by kkossev** [comment]
> I checked on AliExpress most sellers say 24G:

Open a dispute and ask for a full refund.

**Comment by 13717033460** [comment]
Sorry, it has a lot to do with our early technical publicity mistakes, because we did produce 24G products in the early days, and later because of the wrong 24G alarm, the experience is not very good. After 24G matures, 24G radar will be used again

发自我的iPhone


------------------ Original ------------------
From: kkossev ***@***.***&gt;
Date: Tue,Aug 29,2023 11:49 PM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





  
I checked on AliExpress most sellers say 24G:
  
Open a dispute and ask for a full refund.

**Comment by kkossev** [comment]
> Is there any real 24G? - https://www.aliexpress.com/item/1005005947206956.html
> What about this?

This one is real 24Ghz, based on LD2410C mmWave sensor. 
 But it is terribly chatty as the previous TS0601 models (sends the unnecessary distance measurement every single second - non-stop until a presence is sensed).

**Comment by kkossev** [comment]
@13717033460 one very good thing about your new products is that they are not as chatty / not spammy as the rest of the Tuya-based 5.8Ghz radars. This is what made me so enthusiastic initially... until recently.

**Comment by alexvaltchev** [comment]
Sorry to hijack the topic here, we figured it out these square ones are 5.8G. Does anyone have experience finding real 24G?

How about THIS:

Moes - https://www.aliexpress.com/item/1005005932571226.html
![Screenshot_7](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/f6682c27-7f03-4bc0-aa5d-895d9307029c)

**Comment by 13717033460** [comment]
At the beginning of last month, we notified most of the sellers and changed the parameter to 5.8G。


Tomorrow, we'll find out which customer this is, and if you need to return it, I can coordinate.

发自我的iPhone


------------------ Original ------------------
From: kkossev ***@***.***&gt;
Date: Tue,Aug 29,2023 11:49 PM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





  
I checked on AliExpress most sellers say 24G:
  
Open a dispute and ask for a full refund.

**Comment by alexvaltchev** [comment]
@13717033460 what is the detection angel in degree? I put it 2 meters above and here are my settings, and is still not detecting.

![Screenshot_1](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/609fa234-8956-4736-ab1e-925b28240261)

**Comment by 13717033460** [comment]
At a very close distance ，Close to 90 degrees，After setting the parameters, wait 10 seconds。


Please check&nbsp;http://doc.szalarm.com/zg-205Z

发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Wed,Aug 30,2023 0:04 AM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
@13717033460 what is the detection angel in degree? I put it 2 meters above and here are my settings, and is still not detecting.

**Comment by 13717033460** [comment]
The triggering of the device from "NONE" to "Presence" state must first detect large movements in order to trigger radar small movements and static detection;



发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Wed,Aug 30,2023 0:04 AM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
@13717033460 what is the detection angel in degree? I put it 2 meters above and here are my settings, and is still not detecting.

**Comment by alexvaltchev** [comment]
Yes, am aware of it. Thank you. I already read everything there. I have been reading all materials about this sensor, but so far in my case does not work good. Is even worst than the older version on my opinion.

![IMG_20230829_191835](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/419be009-6a46-48e4-b5bc-fd7206d87c20)
![IMG_20230829_191844](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/09ca893b-2555-49b3-9219-3c776fab3520)

It shows HOME from time to time than AWAY again.

**Comment by 13717033460** [comment]
You mean it says Home all the time?



发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Wed,Aug 30,2023 0:20 AM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
Yes, am aware of it. Thank you. I already read everything there. I have been reading all materials about this sensor, but so far in my case does not work good. Is even worst than the older version on my opinion.
 

 
 
It shows HOME from time to time than AWAY again.

**Comment by alexvaltchev** [comment]
@13717033460 
No, the opposite. IT shows presence FALSE, and rarely goes to TRUE for a little time and go back to FALSE. NO presence. The room is 5 meters long and 2.7 meters tall. I put the sensor on 2.1 meters high. I angeled it toward me a little down and still loses me. IS not detecting me.

**Comment by alexvaltchev** [comment]
IF sensitivity is 9 is not detecting. IF sensitivity is 10 is detecting all the time (even when nobody is there)

**Comment by 13717033460** [comment]
How far away are you from him?&nbsp;This side mount product, you can not install so high，and fading time set to 60S


发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Wed,Aug 30,2023 0:36 AM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
IF sensitivity is 9 is not detecting. IF sensitivity is 10 is detecting all the time (even when nobody is there)

**Comment by alexvaltchev** [comment]
@13717033460 I am 1.7 meters from the device. Is mounted 2.1 meters from the ground. Fading time is 10 seconds, I want to react fast not to wait too long. I sit on the computer, I am static but it shows there is nobody, no presence detected.

**Comment by 13717033460** [comment]
You are still, fading must be 40s chapped on, and it is recommended to mount parallel to the person, because this product is side mount, there is no bracket

发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Wed,Aug 30,2023 0:44 AM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
@13717033460 I am 1.7 meters from the device. Is mounted 2.1 meters from the ground. Fading time is 10 seconds, I want to react fast not to wait too long. I sit on the computer, I am static but it shows there is nobody, no presence detected.

**Comment by 13717033460** [comment]
If you are stationary and need to detect breathing action, you will be facing it, within 3m. But most people make tiny movements when they're not asleep,



发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Wed,Aug 30,2023 0:44 AM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
@13717033460 I am 1.7 meters from the device. Is mounted 2.1 meters from the ground. Fading time is 10 seconds, I want to react fast not to wait too long. I sit on the computer, I am static but it shows there is nobody, no presence detected.

**Comment by alexvaltchev** [comment]
@13717033460  so if more than 3 meters it will not detect me if sleeping? In order to detect static need to be less than 3 meters?

Why do I need 40 seconds fade? I want the light to turn off suoer fast l, after 5 seconds after I left the room.

**Comment by alexvaltchev** [comment]
There is stand for the sensor
https://a.aliexpress.com/_mP2Ol6e

**Comment by 13717033460** [comment]
Yes, this is a problem with either 24G or 60G radar. If you are not completely stationary, you can set it for 10 seconds and move the radar as close to the detection area as possible. You can test what you're doing when you're reading a book, when you're working

发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Wed,Aug 30,2023 1:10 AM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
@13717033460  so if more than 3 meters it will not detect me if sleeping? In order to detect static need to be less than 3 meters?
 
Why do I need 40 seconds fade? I want the light to turn off suoer fast l, after 5 seconds after I left the room.

**Comment by 13717033460** [comment]
Sorry,I can't open the link. I'm on my phone. Give me the screenshot



发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Wed,Aug 30,2023 1:15 AM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)





 
There is stand for the sensor
 https://a.aliexpress.com/_mP2Ol6e

**Comment by alexvaltchev** [comment]
![-18011309801406910656](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/99f231eb-a1ea-41c4-9a10-f8ab5622916c)
![-1867896495513142581](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/4171e97f-3b0b-4808-aa84-42ce4dc2467f)
![1669302841-472323002](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/50cc88fb-fbc8-4f92-86e9-5ffb745a6ffa)
![8728549611916970155](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/898829b4-c1a5-40f6-9cac-5325ae9bbd77)

**Comment by 13717033460** [comment]
This is pir



发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Wed,Aug 30,2023 1:25 AM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)

**Comment by 13717033460** [comment]
发自我的iPhone


------------------ Original ------------------
From: Alex Valtchev ***@***.***&gt;
Date: Wed,Aug 30,2023 1:25 AM
To: Koenkk/zigbee2mqtt ***@***.***&gt;
Cc: jomders ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [Koenkk/zigbee2mqtt] [New device support]: TuYa 24GHz HumanPresence Sensor TS0225_TZE200_2aaelwxk (Issue #18612)

**Comment by alexvaltchev** [comment]
![Screenshot_2023-08-29-20-45-42-537_com alibaba aliexpresshd-edit](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/cab922a8-4caf-403c-b83c-2f24a104ebf1)
![-894897513-1512213029](https://github.com/Koenkk/zigbee2mqtt/assets/118940240/c383c52e-55cd-4990-934d-fe5a2c96fbe2)

@13717033460  this is another senaor holder to angel the sensor into the right direction.

**Comment by kkossev** [comment]
@alexvaltchev that's a very good idea (using the Aqara sensor holder) - the precise aiming to the direction where the stationary/breath sensing is expected is of critical importance for achieving the best results.  Precise aiming also helps a lot to avoid interference from ceiling of floor-standing fans, air conditioners, etc.

**Comment by alexvaltchev** [comment]
Yes I already purchased. Waiting on it. I can send you a link for the second one when I get home if you want.

**Comment by kkossev** [comment]
Yes, please.

**Comment by magicmellon** [comment]
Appologies for joining a stale chat - I just wanted to confirm what this device _actually_ is - I have the Zigbee Model: TS0225, Zigbee Manufacturer: _TZE200_2aaelwxk, is this a 5.8GHz or 28GHz device? If 5.8, is it worth updating the Zigbee2MQTT description to reflect that?

**Comment by hepspl** [comment]
> use https://gist.github.com/Koenkk/79a20ea7e5eaf35749e157d0c31a1f9b for TS0225 and _TZE200_hl0ss9oa /sorry

It crashes the daemon :( 

> (node:8920) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'datapoints' of undefined

**Comment by Hammerhand17** [comment]
> One more question. Bought several of these devices, which are now detected as soon as I add them in z2mqtt:
> 
> ![image](https://private-user-images.githubusercontent.com/20372496/286981510-faf78092-f3d1-4f8e-a7aa-be3b77a7bff9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE1MTg4MjYsIm5iZiI6MTcwMTUxODUyNiwicGF0aCI6Ii8yMDM3MjQ5Ni8yODY5ODE1MTAtZmFmNzgwOTItZjNkMS00ZjhlLWE3YWEtYmUzYjc3YTdiZmY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjAyVDEyMDIwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ3OGJhZjdlNzk3ODRkYmNiZmJhMWMzY2E0YzlkMjY0N2ZjZWRhNWMwNWIwYzE3OGIxNmRmZjdmNjE1ZDhiNGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lVfDAKXjoCmqan8YXOhk3GUuF0f-HfwAoGWGl6VwNTg)
> 
> Besides, illuminance value is not detected from Home Assistant:
> 
> ![image](https://private-user-images.githubusercontent.com/20372496/286981732-44a3cda9-0d11-4d37-8b9f-dca5f37f3e64.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE1MTg4MjYsIm5iZiI6MTcwMTUxODUyNiwicGF0aCI6Ii8yMDM3MjQ5Ni8yODY5ODE3MzItNDRhM2NkYTktMGQxMS00ZDM3LThiOWYtZGNhNWYzN2YzZTY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjAyVDEyMDIwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk3OTIwNGYwOTQyYmIzYjg3ZWM3NDdhYmNlOGJiMDI2NTExZTEzMDhmNDhiOGFhNGI3ZGI0OTNkYjViMGY5ODcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HpRxUIbnjPN97t1aipna1kf3T4EXBwbH6PLRl-sYhdc)
> 
> Any clue?

Just in case anyone needs it; just have to go to sensor page and enable this entity.

**Comment by HNKNTA** [comment]
I know this ticket is closed, so I'd appreciate any recommendations on what could I do with the following:
I've ordered a few devices on aliexpress recently, like the ones mentioned on many sources as 5.8GHz device, like this
![image](https://github.com/Koenkk/zigbee2mqtt/assets/6562391/8464f182-c9e0-4d95-8527-77112fd4d18b)
https://www.amazon.com/Presence-Millimeter-Detection-Zigbee2mqtt-Assistant%C2%A1%C2%AD/dp/B0CFHL81P8?th=1

It works currently as _TZE200_2aaelwxk via zigbee2mqtt, described as 5.8Ghz Human presence sensor.

And looking at this issue, I started a dispute on aliexpress, as advised above (and on some other forum) where guys prove this is 5.8GHz device by taking pictures of the disassembled device.

So, I've disassembled mine, as what I see: it's actually 24GHz! 
Revision V2.1, 24000MHz quartz:
![DSC09232](https://github.com/Koenkk/zigbee2mqtt/assets/6562391/4115c489-d9cf-46f5-951a-4f7af996355e)
Antenna:
![DSC09231](https://github.com/Koenkk/zigbee2mqtt/assets/6562391/db9434d2-5340-4e3a-af3f-e05a764fa5d5)

Should I open a new issue for this? So far it works as is, but the illuminance values are crazy high, not sure if it's possible to calibrate it.

**Comment by GitTheFOuttaHere** [comment]
> Should I open a new issue for this? So far it works as is, but the illuminance values are crazy high, not sure if it's possible to calibrate it.

What value do you get & do you have another luminance sensor to compare the values at the same location?

A tuya.valueConverter.divideByxx (xx being the value you set) can be added to an external converter.

So if you have one luminance sensor showing 48 & you have the one with high values showing 480, 4800, 48000 or the likes, you can then use the above to divide by 10, 100, 1000 respectively.

**Comment by HNKNTA** [comment]
> > Should I open a new issue for this? So far it works as is, but the illuminance values are crazy high, not sure if it's possible to calibrate it.
> 
> What value do you get & do you have another luminance sensor to compare the values at the same location?
> 
> A tuya.valueConverter.divideByxx (xx being the value you set) can be added to an external converter.
> 
> So if you have one luminance sensor showing 48 & you have the one with high values showing 480, 4800, 48000 or the likes, you can then use the above to divide by 10, 100, 1000 respectively.

Yeah, I got like 10x values right now in one room, my other luminance sensors also not perfect though. The other room shows 56 vs 2134 on tuya sensor 🫤

I guess you are referring to [this](https://www.zigbee2mqtt.io/guide/configuration/more-config-options.html#network-map), I will try to look into it on weekends, and see what I can do with it, thanks for the help!

**Comment by GitTheFOuttaHere** [comment]
> Yeah, I got like 10x values right now in one room, my other luminance sensors also not perfect though. The other room shows 56 vs 2134 on tuya sensor 🫤
> 
> I guess you are referring to [this](https://www.zigbee2mqtt.io/guide/configuration/more-config-options.html#network-map), I will try to look into it on weekends, and see what I can do with it, thanks for the help!

I have the exact same sensor with same id code. It is automatically recognized in z2mqtt & no external converter is required.

Mine went up to over 3000lm almost every time there was motion. Found it was the led indicator on the front panel causing it.

Turned off, problem solved.

**Comment by GitTheFOuttaHere** [comment]
And as for the external converter, there is some in this thread.

**Comment by HNKNTA** [comment]
@GitTheFOuttaHere I have the LED turned off, but the values are still high.
![image](https://github.com/Koenkk/zigbee2mqtt/assets/6562391/db78e74e-77a5-463e-aaad-22303dac1025)

And it's really higher than it should be, according to the description on the wiki and my other sensors:
https://en.wikipedia.org/wiki/Lux
![image](https://github.com/Koenkk/zigbee2mqtt/assets/6562391/a6eb2615-9747-4734-8bcb-419c46f4c818)


Do you have normally accurate values for your device?

**Comment by GitTheFOuttaHere** [comment]
Depends what "normally accurate" entails. My sensor (specifically TS0225_TZE200_2aaelwxk) would only go higher than 2000lm if I shine a bright led torch directly at it. I only have one of these and have mounted it under a cupboard. I have it there so it does not detect movement from the ceiling fan. And won't pickup my animals most times because there is a bench below it as well. It's also a good way to know how much actual sunlight there is. The values aren't accurate, but it works enough for me to use the readings I'm given.

A led torch I have which has a 1000lm output (on high) shining directly at it, shows over 3000lm.

The photo sensitive component isn't exposed, it's hidden under the plastic case. So not sure how accurate you will need it to be.

If your need for lumens value is critical, you may need a dedicated lux sensor.

Having said that, the lumens value for mine seems to be on par with the wiki. Had mostly overcast days and the value didn't go much higher than 1000lm.

The other sensors I have have different outputs, not in lumens. Their reading goes from 0 to 50 only.

I've ordered another one of these and I'll post the results for you.

**Comment by HNKNTA** [comment]
@GitTheFOuttaHere appreciate your extended response, I guess I need some dedicated sensor to be able to directly compare with my existing ones, integrated with presence and motion sensors. 
Tuya's values seems too high, but according to your response this might be quite accurate, so the other sensors make me doubt as well.

**Comment by GitTheFOuttaHere** [comment]
> @GitTheFOuttaHere appreciate your extended response, I guess I need some dedicated sensor to be able to directly compare with my existing ones, integrated with presence and motion sensors. Tuya's values seems too high, but according to your response this might be quite accurate, so the other sensors make me doubt as well.

Always welcome.

I myself am interested in the values I can use. All my sensors being in different locations give different readings. I find the values required for that area & write the automation to suit. Lights on values, sunny day values, overcast.

When setting up a sensor, I keep track of the brightness in the room via sight, then when it's just too dark I record the values.

You are probably aware of how to set up a sensor, so I'll wait until I get the next unit & compare them for you.

I do have an Aqara luminance sensor, somewhere. I'll dig it out and compare it to that.

Bear in mind I have 2 x Aqara devices with luminance. One is dedicated to luminance only, the other is a aqara pir sensor with built in luminance.

The luminace sensor gives values of around 1000 on daybreak & can go all the way up to 60000 in full sunlight.

But while this sensor might show 900-1200, the pir one shows only 8-13.

**Comment by GitTheFOuttaHere** [comment]
Received the new sensor today, along with a ceiling type (still shows up as _TZE200_2aaelwxk) & all report roughly the same values.

I say "roughly" because they are not all exact, but definitely in the same sort of range.

1st sensor I received: 927
2nd:                          938
Ceiling:                      1028

The first two have the photoresistor behind the shell.

The ceiling one has a hole to let the light in. Which probably explains the different reading.

All tested in the same location.

**Comment by MichalLytek** [comment]
> I know this ticket is closed, so I'd appreciate any recommendations on what could I do with the following: I've ordered a few devices on aliexpress recently, like the ones mentioned on many sources as 5.8GHz device, like this
> 
> It works currently as _TZE200_2aaelwxk via zigbee2mqtt, described as 5.8Ghz Human presence sensor.
> 
> And looking at this issue, I started a dispute on aliexpress, as advised above (and on some other forum) where guys prove this is 5.8GHz device by taking pictures of the disassembled device.
> 
> So, I've disassembled mine, as what I see: it's actually 24GHz! Revision V2.1, 24000MHz quartz: 

This is 24 MHz quartz, used by the TeLink MCU:
https://wiki.telink-semi.cn/doc/ds/PB_TLSR8258-E_Product%20Brief%20for%20Telink%20BLE%20IEEE802.15.4%20Multi-Standard%20Wireless%20SoC%20TLSR8258.pdf

Not 24 000 MHz for the radar - you need unsolder the second pcb with the mmwave sensor to make sure it's 5.8 or 24 GHz.

**Comment by MichalLytek** [comment]
> I know this ticket is closed, so I'd appreciate any recommendations on what could I do with the following: I've ordered a few devices on aliexpress recently, like the ones mentioned on many sources as 5.8GHz device, like this
> 
> It works currently as _TZE200_2aaelwxk via zigbee2mqtt, described as 5.8Ghz Human presence sensor.
> 
> And looking at this issue, I started a dispute on aliexpress, as advised above (and on some other forum) where guys prove this is 5.8GHz device by taking pictures of the disassembled device.
> 
> So, I've disassembled mine, as what I see: it's actually 24GHz! Revision V2.1, 24000MHz quartz: 

@HNKNTA

This is 24 MHz quartz, used by the TeLink MCU:
https://wiki.telink-semi.cn/doc/ds/PB_TLSR8258-E_Product%20Brief%20for%20Telink%20BLE%20IEEE802.15.4%20Multi-Standard%20Wireless%20SoC%20TLSR8258.pdf

Not 24 000 MHz for the radar - you need to unsolder the second pcb with the mmwave sensor to make sure it's 5.8 or 24 GHz.

**Comment by GitTheFOuttaHere** [comment]
How could someone confuse 24.000mhz with 24ghz. Maybe they didn't see the dot (.) in 24.000 mhz?

There is 24ghz version & 5.8ghz version. Some sellers pass off the 5.8g as 24g.

**Comment by hknkbp** [comment]
Hi, first time poster on git. I have bought the same sensor as @HNKNTA and Z2M identifies it wrongly as ZG-205Z/A 5.8Ghz Human presence sensor. This is definitely different as I've seen multiple pictures of the /A board. I've opened it and see "ZG-205Z v2.1" written on the board. 
![20240314_112738](https://github.com/Koenkk/zigbee2mqtt/assets/93479870/d2f05490-67fe-4e65-b7ab-040bf6cec7bb)
![20240314_112544](https://github.com/Koenkk/zigbee2mqtt/assets/93479870/1a707c76-8a99-413f-83b3-f09b7e13866d)
I'm no expert, but the board looks more like a 24Ghz than the 5.8Ghz. I also have a ZY-M100-L.
![Screenshot_20240316_230858_Home Assistant](https://github.com/Koenkk/zigbee2mqtt/assets/93479870/166edfe7-6e80-4a57-849b-33cb175f61da)
I would've liked to see distance etc on the ZG-205Z too.
![Screenshot_20240316_230805_Home Assistant](https://github.com/Koenkk/zigbee2mqtt/assets/93479870/208df28d-d954-489a-a724-e574a7fdb823)

**Comment by Hammerhand17** [comment]
> Hi!
> 
> I also have a pair of ZG-205Z v2.1 that I bought to replace two ZY-M100-24G that were saturating the Zigbee network with 3-4 messages every second, each.
> 
> These new ones send much fewer messages, but even though Z2M does show the illuminance value, I can't use the sensor value for automations/scripts. It´s not available in the entity states.
> 
> Is there any way to fix this?
> 
> Also, as @hknkbp says, it would be great to see (an be able to use in automations) the distance, if possible.

For me it was disabled, so I went to sensors, pressed "+8 entities not shown" and then enabled Iluminance. However, mine were ZY-M100-24G, so don't know if those new ones of yours will suffer the same.

**Comment by elmaswebon** [comment]
> For me it was disabled, so I went to sensors, pressed "+8 entities not shown" and then enabled Iluminance. However, mine were ZY-M100-24G, so don't know if those new ones of yours will suffer the same.

Yep, that was it! Thanks!

**Comment by bapesupreme** [comment]
> > Yeah, I got like 10x values right now in one room, my other luminance sensors also not perfect though. The other room shows 56 vs 2134 on tuya sensor 🫤
> > I guess you are referring to [this](https://www.zigbee2mqtt.io/guide/configuration/more-config-options.html#network-map), I will try to look into it on weekends, and see what I can do with it, thanks for the help!
> 
> I have the exact same sensor with same id code. It is automatically recognized in z2mqtt & no external converter is required.
> 
> Mine went up to over 3000lm almost every time there was motion. Found it was the led indicator on the front panel causing it.
> 
> Turned off, problem solved.

how did you turn off the led indicator? was a tuya.valueConverter.divideByxx ever added to the converter for this device?

I am on 2.1.0 and it seems to be reporting the raw illuminance lux value

I changed my setting based on what I found here and will be testing
https://github.com/Koenkk/zigbee2mqtt/issues/23478#issuecomment-2265201485
